doc/source/rayserve/overview.rst

```diff
@@ -0,0 +1,197 @@
+RayServe: Scalable and Programmable Serving
+===========================================
+
+.. image:: logo.svg
+    :align: center
+    :height: 250px
+    :width: 400px
+
+Overview
+--------
+
+RayServe is a scalable model-serving library built on Ray Core.
+
+RayServe allows you to:
+
+- **Framework Agnostic**:Use the same toolkit to serve everything from deep learning models 
+  built with frameworks like PyTorch or TensorFlow to scikit-learn models or arbitrary business logic.
+- **Simplify MLOps**: Configure your model serving with pure Python code - no more YAMLs or 
+  JSON configs.
+-  **A/B test models** (with zero downtime) by decoupling routing logic from response handling logic.
```

Comment:
The bullet points 3 and 4 seem discontinuous from 1 and 2 btw

Reply:
Added some more context.